api-reference/_hidden/GridBaseColumn/calculateCellValue.md

```diff
@@ -13,7 +13,7 @@ The data of the row to which the cell belongs.
 A cell's custom value.
 
 ---
-Unlike data columns bound to a [data field](/api-reference/_hidden/GridBaseColumn/dataField.md '{basewidgetpath}/Configuration/columns/#dataField'), unbound columns display custom values returned from the **calculateCellValue** function. It is called each time a new row is rendered.
+Unlike data columns bound to a [data field](/api-reference/_hidden/GridBaseColumn/dataField.md '{basewidgetpath}/Configuration/columns/#dataField'), unbound columns display custom values returned from the **calculateCellValue** function. The component executes this function multiple times for each record: when records are rendered, when users sort or filter then, and when summaries are computed. To avoid errors and enhance the UI component performance, make sure that properties of the **rowData** object used in calculation exist and keep calculations inside this function as simple as possible.
```

Comment:
```suggestion
Unlike data columns bound to a [data field](/api-reference/_hidden/GridBaseColumn/dataField.md '{basewidgetpath}/Configuration/columns/#dataField'), unbound columns display custom values returned from the **calculateCellValue** function. The component executes this function multiple times for each record: when records are rendered, when users sort or filter them, and when summaries are computed. To avoid errors and enhance the UI component performance, make sure that properties of the **rowData** object used in calculation exist and keep calculations inside this function as simple as possible.
```